### DISCUSSION TITLE
Problems updating to the latest version of the Docker image.

### DISCUSSION THREAD

**Discussion by EstevoVazquez** [stepci/stepci#137] [Q&A] [answered]
I have been using the Docker image version stepci:2.6.4 without any issues by launching the container in the following way:

`docker run -t --rm \
    -v "$(pwd)"/tests:/tests \
    ghcr.io/stepci/stepci:2.6.4 tests/workflows/api_test_workflow.yml \
    -e host=$HOST -e token=$ACCESS_TOKEN `

Passing the host and JWT as environment variables.

I have the following component defined.

`components:
  credentials:
    jwt:
      bearer:
        token: ${{env.token}}
`

and i use it in test like: 
`      - id: getResource
        name: Get Resource
        http:
          auth:
            $ref: "#/components/credentials/jwt"
          url: https://${{env.host}}/api/v1/resources
          method: GET
          check:
            status: 200
`

When I run the tests with: 

`docker run -t --rm \
    -v "$(pwd)"/tests:/tests \
    ghcr.io/stepci/stepci:latest tests/workflows/api_test_workflow.yml \
    -e host=$HOST -e token=$ACCESS_TOKEN `

It seems that the environment variables are not being passed correctly, and during the test execution, I get the error
`getaddrinfo EAI_AGAIN undefined` ". Has there been any change in how the environment variables are read in the new image? Any help would be appreciated.

**Comment by mishushakov** [comment]
Interesting. Will see what we could do!

**Reply by mishushakov** [reply]
There were some changes in the Docker build indeed

Could you try: `docker run -t --rm \ -v "$(pwd)"/tests:/tests \ ghcr.io/stepci/stepci:2.6.4 tests/workflows/api_test_workflow.yml \ ENV host=$HOST token=$ACCESS_TOKEN`

Else, you could build your own Docker image using the following Dockerfile for now:

```dockerfile
FROM node:lts-alpine
RUN npm install -g stepci
ENTRYPOINT ["stepci", "run"]
```

**Reply by mishushakov** [reply, marked as answer]
Just pushed an update (2.6.7) which resolves that. Sorry for the inconvenience!

**Reply by EstevoVazquez** [reply]
i test it. Everything works fine with 2.6.4 image, but using  stepci:latest  or  stepci:2.6.5 images   I get the error
"getaddrinfo EAI_AGAIN undefined ".  
I think that starting from image 2.6.5, the runner's version was updated. Could that be where the problem lies?

**Comment by EstevoVazquez** [comment]
Tested with 2.6.7 image.  It works like charm !!  Thank you  @mishushakov

**Reply by mishushakov** [reply]
Happy to hear that! Feel free to let me know if you have any other issues/suggestions ☺️
